config/workflow_config_example.json

```diff
@@ -61,6 +61,12 @@
         "parameter_type": "datetime",
         "key_name": "datetime_example_with_default",
         "default": "2023-12-31T20:00:00"
+      },
+      {
+        "parameter_type": "duration",

```

Comment:
Maybe I should have mentioned in an earlier PR: a default of milliseconds seems quite small to me, since most simulations run with 15 min or larger timesteps. This will create a high number as input. I would prefer seconds as default.

Reply:
Incoming from protocol & SDK: https://github.com/Project-OMOTES/omotes-sdk-python/pull/72